smallLibraries/Google.Cloud.Templates/content/gcpmvc/CSharp/.template.config/template.json

```diff
@@ -0,0 +1,121 @@
+{
+  "$schema": "http://json.schemastore.org/template",
+  "author": "Google Cloud Tools",
+  "groupIdentity": "GCP.ASP.NET.MVC",
+  "identity": "GCP.ASP.NET.MVC.CSharp",
+  "name": "Google ASP.NET Core MVC",
+  "shortName": "gcpmvc",
+  "description": "ASP.NET Core MVC template for Google Cloud",
+  "classifications": [ "GCP", "Web", "ASP.NET", "MVC" ],
+  "defaultName": "MyGoogleCloudMvcApp",
```

Comment:
Can we be consistent between "GoogleCloud" and "Gcp"? One of

- "MyGcpMvcApp" and "MyGcpMvcProject"
- "MyGoogleCloudMvcApP" and "MyGoogleCloudMvcProject"


(Admittedly I don't know what these values do...)